requirements/edx/github.txt

```diff
@@ -48,7 +48,7 @@
 
 # Third-party:
 git+https://github.com/jazzband/django-pipeline.git@d068a019169c9de5ee20ece041a6dea236422852#egg=django-pipeline==1.5.3
--e git+https://github.com/edx/django-wiki.git@b232cff63c5d342a5fe8f11a25f03803c7c45d65#egg=django-wiki
+-e git+https://github.com/edx/django-wiki.git@c095baa93bf92427d4da0a61ce8c7775f6b8e286#egg=django-wiki
```

Comment:
Darn! Wish I'd remembered to tell you to bump the version in the django-wiki PR. I'd prefer that a tag went here instead of a SHA.

Reply:
But - honestly - this is just a nit.

Reply:
I gave it some thought, but we've never touched the version for django-wiki and it seemed more weird to change it now or try to come up with a naming/versioning scheme for tags than to leave it as-is. At least for this change.

Reply:
I think using a commit is fine but can we push a tag for the commit on the django-wiki side so it can't disappear in the future?

Reply:
perhaps `release-2017-12-04`

Reply:
I was wrong about this in that we've been doing tagged releases on there, so I'll update this to reflect the new v0.0.15 tag/release that I just created.

Reply:
Or, actually I was entirely wrong and we've been doing version bumps over there and I was just looking at the master branch. So I'm going to do a version bump on edx/django-wiki to v0.0.16, do another release there, then change this to use the release tag. Sorry for the confusion.